src/Sarif.Multitool.Library/Rules/RuleResources.Designer.cs

```diff
@@ -78,6 +78,30 @@ internal class RuleResources {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to The &apos;$schema&apos; property must refer to the final version of the SARIF 2.1.0 schema. This enables IDEs to provide Intellisense for SARIF log files.
+        ///
```

Comment:
Should "The" be capitalized?

Reply:
"Looks up a localized string similar to" is added by VS for use in a tooltip. It's not part of the resource string.